### ISSUE TITLE
Accessibility: switch control fails colour contrast: WCAG 1.4.3

### ISSUE THREAD

**Issue by chris-gds** [chakra-ui/chakra-ui#8020]
### Description

When I visit the switch component page, I expect the control meet WCAG 2.1 AA standards, however the colour does not pass contrast checks.

https://chakra-ui.com/docs/components/switch/usage

The switch:

![switch](https://github.com/chakra-ui/chakra-ui/assets/71266765/50dfd839-8c10-423b-8241-9b35bd585078)

The check:
![colour-check](https://github.com/chakra-ui/chakra-ui/assets/71266765/b082ee35-9bda-4894-9224-c0d87eb514ce)

### Link to Reproduction

https://chakra-ui.com/docs/components/switch/usage

### Steps to reproduce

Go to the [switch page](https://chakra-ui.com/docs/components/switch/usage) and review the colour.

### Chakra UI Version

Latest

### Browser

all

### Operating System

- [X] macOS
- [X] Windows
- [X] Linux

**Comment by segunadebayo** [comment]
This is a design flaw in v2. we've improved this in v3
